core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -141,7 +141,9 @@ public void cancel() { }
     public void halfClose() { }
 
     @Override
-    public void sendPayload(Object payload) { }
+    public boolean sendPayload(Object payload) {
+      return true;
```

Comment:
Right now `isReady()` returns `false` if the stream is closed. Thus, I'm saying that this should be `return false;`. There is no need for a condition.

If `return false;` seems wrong, then `isReady()` should be changed to return `true` if the stream is closed.